tasks/functions/functions.js

```diff
@@ -53,7 +66,16 @@ function getGoogle(n) {
  * }
  */
 function getUser(firstName, lastName, age) {
-
+	let person = {};
```

Comment:
Could you please check the link above about the function's default params and update your function.
https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Functions/Default_parameters